sig-services/service-requirements/contract-duration.md

```diff
@@ -0,0 +1,24 @@
+---
+title: contract duration
+persona: service consumer, service provider/owner
+phase: service-operational
+---
+
+This requirement states that a service should have a defined contract duration. Contract duration states the start and the end date of the subscription to the service or that the subscription doesn't have an end date.
+
+If the contract duration states the end date of the subscription it should also state whether the subscription can be extended. If it can, it should also state if the subscription is going to auto extend without the consumers acknowledgement.
```

Comment:
```suggestion
If the contract duration states the end date of the subscription it should also state whether the subscription can be extended. If it can, it should also state if the subscription is going to auto extend without the consumers explicit acknowledgement.
```